docs/NexusIQServer_Vertical.svg

```diff
@@ -0,0 +1 @@
18,4.53a4.86,4.86,0,0,1-.71,2.62l-1.51-1.78-2,1.68L33,25.2a4.38,4.38,0,0,1-1.43.23m0,2.6a8,8,0,0,0,3.25-.68L36,28.62,38,27l-1-1.17a7.22,7.22,0,0,0,1.79-4.87,6.91,6.91,0,0,0-7.19-7.12,6.92,6.92,0,0,0-7.22,7.12A6.92,6.92,0,0,0,31.61,28M22.74,14H19.8V27.78h2.94Z"/><polygon class="cls-2" points="30.18 5.34 43.26 12.88 43.26 28.97 29.3 37.01 15.35 28.97 15.35 13.99 10.8 11.37 10.8 31.59 29.3 42.25 47.8 31.59 47.8 10.26 30.18 0.1 30.18 5.34"/><polygon class="cls-1" points="28.36 0 11.54 9.7 16.08 12.32 28.36 5.23 28.36 0"/><path d="M14.5,53.55c0-1.4-.72-1.86-1.81-1.86a3.06,3.06,0,0,0-2.34,1.23V59H8.77v-8.5h1.58v1.2a4.06,4.06,0,0,1,3-1.41A2.44,2.44,0,0,1,16.09,53v6H14.5Z"/><path d="M21.28,50.28c2.55,0,4.14,2,4.14,4.59v.39H18.7a2.78,2.78,0,0,0,2.88,2.64,3.63,3.63,0,0,0,2.52-1l.73,1a4.76,4.76,0,0,1-3.41,1.27,4.46,4.46,0,0,1-.14-8.92Zm-2.6,3.84h5.19a2.52,2.52,0,0,0-2.59-2.54A2.6,2.6,0,0,0,18.68,54.12Z"/><path d="M29.48,55.68,27.09,59H25.3l3.18-4.36-3-4.14h1.79l2.2,3.06,2.19-3.06h1.79l-3,4.14L33.65,59H31.86Z"/><path d="M40.09,57.83a4.2,4.2,0,0,1-3,1.37,2.4,2.4,0,0,1-2.71-2.71v-6h1.58V56c0,1.41.71,1.83,1.8,1.83a3.06,3.06,0,0,0,2.35-1.19V50.49h1.59V59H40.09Z"/><path d="M43.51,56.72a4.13,4.13,0,0,0,2.85,1.21c1.2,0,1.83-.49,1.83-1.21s-1-1.13-2.06-1.37C44.69,55,43,54.63,43,52.8c0-1.36,1.16-2.52,3.24-2.52a4.6,4.6,0,0,1,3.24,1.21l-.71,1.11a3.32,3.32,0,0,0-2.51-1c-1.08,0-1.75.42-1.75,1.1s.92,1,2,1.23c1.47.34,3.23.74,3.23,2.7,0,1.46-1.19,2.6-3.43,2.6a4.86,4.86,0,0,1-3.54-1.32Z"/><path class="cls-3" d="M0,72V63H1.58v9Z"/><path class="cls-3" d="M2.56,67.49a4.48,4.48,0,0,1,4.61-4.66,4.48,4.48,0,0,1,4.6,4.66,4.68,4.68,0,0,1-1.2,3.24l.74.84-1.09.92-.78-.88a4.82,4.82,0,0,1-2.27.54A4.47,4.47,0,0,1,2.56,67.49Zm5.88,3L7.32,69.21l1.1-.92,1.13,1.27a3.56,3.56,0,0,0,.6-2.07,3,3,0,1,0-3,3.26A3.07,3.07,0,0,0,8.44,70.48Z"/><path class="cls-3" d="M15.84,69.72a4.13,4.13,0,0,0,2.85,1.21c1.2,0,1.83-.49,1.83-1.21s-1-1.13-2.06-1.37C17,68,15.33,67.63,15.33,65.8c0-1.36,1.16-2.52,3.24-2.52a4.55,4.55,0,0,1,3.23,1.21l-.7,1.11a3.33,3.33,0,0,0-2.52-1c-1.07,0-1.74.42-1.74,1.1s.92,1,2,1.23c1.48.34,3.24.74,3.24,2.7,0,1.46-1.2,2.6-3.43,2.6a4.86,4.86,0,0,1-3.54-1.32Z"/><path class="cls-3" d="M26.73,63.28c2.55,0,4.14,2,4.14,4.59v.39H24.15A2.79,2.79,0,0,0,27,70.9a3.61,3.61,0,0,0,2.52-1l.74,1a4.78,4.78,0,0,1-3.42,1.27,4.46,4.46,0,0,1-.14-8.92Zm-2.6,3.84h5.19a2.52,2.52,0,0,0-2.59-2.54A2.6,2.6,0,0,0,24.13,67.12Z"/><path class="cls-3" d="M31.8,63.49h1.59v1.3a3.61,3.61,0,0,1,2.78-1.49v1.6a2.29,2.29,0,0,0-.55-.05,3,3,0,0,0-2.23,1.23V72H31.8Z"/><path class="cls-3" d="M36.5,63.49h1.69l2.66,6.67,2.65-6.67h1.71L41.71,72H40Z"/><path class="cls-3" d="M49.17,63.28c2.55,0,4.14,2,4.14,4.59v.39H46.59a2.79,2.79,0,0,0,2.88,2.64,3.63,3.63,0,0,0,2.52-1l.74,1a4.78,4.78,0,0,1-3.42,1.27,4.46,4.46,0,0,1-.14-8.92Zm-2.6,3.84h5.19a2.52,2.52,0,0,0-2.59-2.54A2.6,2.6,0,0,0,46.57,67.12Z"/><path class="cls-3" d="M54.24,63.49h1.59v1.3a3.61,3.61,0,0,1,2.78-1.49v1.6a2.29,2.29,0,0,0-.55-.05,3,3,0,0,0-2.23,1.23V72H54.24Z"/></g></g></svg>
```

Comment:
do these SVG files need to be referenced anywhere in the project, like in an MD file or will they be referenced later?

Reply:
@adrianpowell SVG files are referenced from /charts/nexus-iq/Chart.yaml and /charts/nexus-repository-manager/Chart.yaml